components/purchase_modal/purchase_modal.tsx

```diff
@@ -596,63 +480,30 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
-                        </div>
-                        {this.state.selectedProduct?.billing_scheme === BillingSchemes.FLAT_FEE &&
-                            <Badge className='unlimited-users-badge'>
-                                <FormattedMessage
-                                    defaultMessage={'Unlimited Users'}
-                                    id={'admin.billing.subscription.unlimitedUsers'}
-                                />
-                            </Badge>
-                        }
-                        {priceSection}
-                        <button
-                            disabled={!this.state.paymentInfoIsValid || this.state.selectedProduct?.billing_scheme === BillingSchemes.SALES_SERVE}
-                            onClick={this.handleSubmitClick}
-                        >
-                            {buttonTitle}
-                        </button>
-                        <div className='fineprint-text'>
-                            <span>
-                                {bottomInformationMsg}
-                            </span>
-                            {'\u00A0'}
-                            <a
-                                href={CloudLinks.BILLING_DOCS}
-                                target='_blank'
-                                rel='noopener noreferrer'
-                            >
-                                <FormattedMessage
-                                    defaultMessage={'See how billing works.'}
-                                    id={'admin.billing.subscription.howItWorks'}
-                                />
-                            </a>
-                        </div>
-                    </div>
-                    <div className='footer-text'>
-                        <FormattedMessage
-                            defaultMessage={'Need other billing options?'}
-                            id={'admin.billing.subscription.otherBillingOption'}
-                        />
-                    </div>
-                    {this.contactSalesLink(
-                        <FormattedMessage
-                            defaultMessage={'Contact Sales'}
-                            id={
-                                'admin.billing.subscription.privateCloudCard.contactSales'
-                            }
-                        />,
-                    )}
-                    <div className='logo'>
-                        <MattermostCloudSvg
-                            width={250}
-                            height={28}
-                        />
+                    <div className='plan_comparison'>
+                        {this.comparePlan}
                     </div>
+                    <Card
+                        topColor='#4A69AC'
+                        plan={this.state.selectedProduct ? this.state.selectedProduct.name.split(' ')[1] : ''}
+                        price={`$${this.state.selectedProduct?.price_per_seat.toString()}`}
```

Comment:
Same here, a `null` check to be safe
```suggestion
                        price={`$${this.state.selectedProduct?.price_per_seat?.toString()}`}
```